PWGJE/FlavourJetTasks/AliAnalysisTaskHFSubstructure.cxx

```diff
@@ -1373,15 +1368,14 @@ Bool_t AliAnalysisTaskHFSubstructure::FillHistograms()
     if (NTruthD==5) fhEvent->Fill(16); 
     if (NTruthD==6) fhEvent->Fill(17);
     }
-    delete Particle_Container; 
+    //delete Particle_Container; 
   }
 
-
-  
- return kTRUE;
-
+  return kTRUE;
 
 
+  delete fFastJetWrapper_Truth;
+  delete fFastJetWrapper;
```

Comment:
These lines happen after a return statement so they are without effect.

Reply:
Hi markus, yes sorry I meant to put them above, just a typo. Thanks for pointing it out

Reply:
OK, please fix and I will approve the PR.